supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -67,3 +67,834 @@ For documentation questions, contact rhel-docs@redhat.com.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[greenboot]]
+==== image:images/yes.png[yes] greenboot (noun)
+*Description*: Generic Health Check Framework for systemd on rpm-ostree based systems.
+
+*Use it*: yes
+
+*Incorrect forms*:  Greenboot, green boots
+
+*See also*:
+
+[discrete]
+[[gssapi]]
+==== image:images/yes.png[yes] GSSAPI (noun)
+*Description*: The Generic Security Service Application Program Interface (GSSAPI, or GSS-API) allows developers to abstract how their applications protect data that is sent to peer applications. Security-service vendors can provide GSSAPI implementations of common procedure calls as libraries with their security software. These libraries present a GSSAPI-compatible interface to application writers who can write their application to use only the vendor-independent GSSAPI. With this flexibility, developers do not have to tailor their security implementations to any particular platform, security mechanism, type of protection, or transport protocol.
+
+Kerberos is the dominant GSSAPI mechanism implementation, which allows Red Hat Enterprise Linux and Microsoft Windows Active Directory Kerberos implementations to be API compatible.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:kerberos[kerberos]
+
+[discrete]
+[[hidden-replica]]
+==== image:images/yes.png[yes] hidden replica (noun)
+*Description*: A hidden replica is an IdM replica that has all services running and available, but its server roles are disabled, and clients cannot discover the replica because it has no SRV records in DNS.
+
+Hidden replicas are primarily designed for services such as backups, bulk importing and exporting, or actions that require shutting down IdM services. Since no clients use a hidden replica, administrators can temporarily shut down the services on this host without affecting any clients. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/planning_identity_management/planning-the-replica-topology_{ProjectNameID}#the-hidden-replica-mode_planning-the-replica-topology[The hidden replica mode].
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:dns-srv-records[dns-srv-records]
+
+[discrete]
+[[host-system]]
+==== image:images/yes.png[yes] host system (noun)
+*Description*: The system on which the instrumentation modules (from SystemTap scripts) are compiled, to be loaded on target systems.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:target-system[target-system]
+
+[discrete]
+[[http-server]]
+==== image:images/yes.png[yes] HTTP server (noun)
+*Description*: See xref:web-server[web-server]
```

Comment:
I'm not sure if we do this elsewhere in the guide. I think that this entry should have its own description, and then link to "web server" in the "See also" section

Reply:
I leave it out since the meaning is the same as web server